pkg/plugins/pfs/grafanaplugin.cue

```diff
@@ -0,0 +1,29 @@
+package pfs
```

Comment:
It does make sense to have this here, in terms of `pkg/plugins` being the space where most plugin things get defined. But it's really kinda properly a part of kindsys, at minimum because we can't really evaluate changes to `Composable` over there without looking at this, as well.

Reply:
No strong opinions. But felt earlier when moving everything plugin related under plugins package we wanted the kind package to be as generic as possible without any details of how plugins are being found/parsed. Is it possible to move some stuff and keep some perhaps?